composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "a003c15d0428178d28dd954a18766ec9",
+    "content-hash": "3052afdac31561cdc43064a657f79b64",
```

Comment:
This will be an issue with merge-up operations: unsure if I should first merge up `2.20.0`, or merge this and then fix it at merge time.

Reply:
…actually, I've squashed and removed the commit that changes the lock file. No checks will run because no PHP code has changed but once 2.20 is in, everything should be groovy. If it's easier, I can just merge 2.20 into this pull